### ISSUE TITLE
Images.jl incompatible with ImageMorphology@0.4.1 

### ISSUE THREAD

**Issue by DominikZians** [JuliaImages/Images.jl#1013]
Can't install Images along ImageMorphology version 0.4.1

Minimum example:
```
(@v1.7) pkg> generate test

(@v1.7) pkg> activate test

(test) pkg> add ImageMorphology@0.4.1
  
(test) pkg> pin ImageMorphology

(test) pkg> add Images
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package ImageMorphology [787d08f9]:
 ImageMorphology [787d08f9] log:
 ├─possible versions are: 0.1.0-0.4.1 or uninstalled
 ├─restricted to versions * by test [ff4e7b30], leaving only versions 0.1.0-0.4.1
 │ └─test [ff4e7b30] log:
 │   ├─possible versions are: 0.1.0 or uninstalled
 │   └─test [ff4e7b30] is fixed to version 0.1.0
 ├─restricted to versions 0.4.1 by an explicit requirement, leaving only versions 0.4.1
 └─restricted by compatibility requirements with Images [916415d5] to versions: 0.1.1-0.3.2 — no versions left
   └─Images [916415d5] log:
     ├─possible versions are: 0.17.3-0.25.2 or uninstalled
     └─restricted to versions * by an explicit requirement, leaving only versions 0.17.3-0.25.2
```

**Comment by DominikZians** [comment]
N.B. ImageSegmantations.jl is also incompatible with ImageMorphology@0.4.1

**Comment by johnnychen94** [comment]
Yeap, this is deliberately made so because otherwise, we'll need to bump Images v0.26. https://github.com/JuliaImages/Images.jl/pull/1011#issuecomment-1158671434

The current ImageMorphology@0.4 version is more or less a "preview" version for people who care about morphology processing. For common Images users that don't care much about morphology, it's better to wait ImageMorphology@0.5 (a version that will match MATLAB's functionality); by then I'll bump Images v0.26 for it.

**Comment by timholy** [comment]
Closed by #1031
